tests/org.jboss.tools.fuse.ui.bot.test/src/org/jboss/tools/fuse/ui/bot/test/FeaturesServerTest.java

```diff
@@ -0,0 +1,105 @@
+package org.jboss.tools.fuse.ui.bot.test;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+
+import org.jboss.reddeer.common.logging.Logger;
+import org.jboss.reddeer.common.wait.AbstractWait;
+import org.jboss.reddeer.common.wait.TimePeriod;
+import org.jboss.reddeer.common.wait.WaitUntil;
+import org.jboss.reddeer.common.wait.WaitWhile;
+import org.jboss.reddeer.core.condition.ShellWithTextIsAvailable;
+import org.jboss.reddeer.core.handler.ShellHandler;
+import org.jboss.reddeer.eclipse.wst.server.ui.view.ServersView;
+import org.jboss.reddeer.junit.runner.RedDeerSuite;
+import org.jboss.reddeer.requirements.cleanworkspace.CleanWorkspaceRequirement.CleanWorkspace;
+import org.jboss.reddeer.requirements.openperspective.OpenPerspectiveRequirement.OpenPerspective;
+import org.jboss.reddeer.swt.impl.button.PushButton;
+import org.jboss.reddeer.swt.impl.button.RadioButton;
+import org.jboss.reddeer.swt.impl.text.DefaultText;
+import org.jboss.tools.common.reddeer.FileUtils;
+import org.jboss.tools.fuse.reddeer.perspectives.FuseIntegrationPerspective;
+import org.jboss.tools.runtime.reddeer.preference.FuseServerRuntimePreferencePage;
+import org.jboss.tools.runtime.reddeer.preference.JBossRuntimeDetection;
+import org.jboss.tools.runtime.reddeer.wizard.DownloadRuntimesWizard;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Class contains test cases on a variety of server feature requests
+ * 
+ * @author djelinek
+ */
+@CleanWorkspace
+@OpenPerspective(FuseIntegrationPerspective.class)
+@RunWith(RedDeerSuite.class)
+public class FeaturesServerTest extends DefaultTest {
+
+	protected Logger log = Logger.getLogger(CamelEditorTest.class);
+
+	@Before
+	public void clean() {
+		// clean previous runtime installations
+		FileUtils.deleteDir(new File(getPath() + "target/temp_install/jboss-fuse-6.3.0.redhat-187"));
+		// close all shells
+		ShellHandler.getInstance().closeAllNonWorbenchShells();
+	}
+
+	/**
+	 * <p>
+	 * Fuse runtime detection never makes a server
+	 * <p>
+	 * <b>Link: </b>
+	 * <a href="https://issues.jboss.org/browse/FUSETOOLS-2176">https://issues.jboss.org/browse/FUSETOOLS-2176</a>
+	 */
+	@Test
+	public void issue_2176() {
```

Comment:
Please use a better name for the test method. The issue number can be used in the Java Doc